pp/integrations/plugin-packs/procedures/applications-veeam-nrpe.md

```diff
@@ -216,9 +216,9 @@ yum install centreon-nrpe3-plugin
 
 | Macro           | Description                                                                                                                                                                                                                      | Default value                                       | Mandatory   |
 |:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------|:-----------:|
-| FILTERENDTIME   | Filter job with end time greater than current time less value in seconds (default: 86400)                                                                                                                                        | 86400                                               |             |
+| FILTERENDTIME   | Tolerance value in seconds, to avoid skipping jobs whose end time is earlier than the current time                                                                                                                                    | 86400                                               |             |
 | FILTERNAME      | Filter job name (can be a regexp)                                                                                                                                                                                                |                                                     |             |
-| FILTERSTARTTIME | Filter job with start time greater than current time less value in seconds                                                                                                                                                       |                                                     |             |
+| FILTERSTARTTIME | olerance value in seconds, to avoid skipping jobs whose start time is earlier than the current time                                                                                                                                                      |                                                     |             |
```

Comment:
```suggestion
| FILTERSTARTTIME | Tolerance value in seconds, to avoid skipping jobs whose start time is earlier than the current time                                                                                                                                                      |                                                     |             |
```